pkg/pxc/service.go

```diff
@@ -20,6 +20,9 @@ func NewServicePXC(cr *api.PerconaXtraDBCluster) *corev1.Service {
 			Labels: map[string]string{
 				"app.kubernetes.io/name":     "percona-xtradb-cluster",
 				"app.kubernetes.io/instance": cr.Name,
+				"app.kubernetes.io/component":  "pxc",
+				"app.kubernetes.io/managed-by": "percona-xtradb-cluster-operator",
+				"app.kubernetes.io/part-of":    "percona-xtradb-cluster",
```

Comment:
could you please add new lables only if `if cr.CompareVersionWith("1.9.0") >= 0 {`
see example in line 44 below
reason - don't trigger changes for all clusters during operator upgrade

Reply:
Alright, I've fixed this for all 4 services. Thanks! :)